src/fiatconnect/slice.ts

```diff
@@ -22,6 +28,8 @@ export interface State {
   quotesError: string | null
   transfer: FiatConnectTransfer | null
   providers: FiatConnectProviderInfo[] | null
+  recentlyUsedFiatAccounts: CachedFiatAccountUse[]
+  attemptReturnUserFlowLoading: boolean
```

Comment:
should this be set to false on rehydrate?

Reply:
yep!